caddy/caddy.go

```diff
@@ -77,6 +79,16 @@ func parse(c *caddy.Controller) (txtdirect.Config, error) {
 			}
 			resolver = resolverAddr[0]
 
+		case "proxy":
+			toProxy := c.RemainingArgs()
+			if len(toProxy) != 1 {
```

Comment:
Not sure, why you want to check for this specific length. Could you give a bit more context on what you wanted to do?

Reply:
I was thinking I could check if the proxy config contains 1 token as a starting point with a simple example. I changed this to return an error if the proxy arguments in the config file are empty instead.